molecules/_tables.scss

```diff
@@ -51,37 +51,41 @@ table {
     border-bottom: 1px solid $table-header-border-color;
     background-color: $table-header-background-color;
     color: $table-header-color;
-    th {
-      text-align: left;
-    }
-    tr:not(:last-child) th {
-      border-bottom: 1px solid $table-header-row-border-bottom-color;
-    }
-    tr:not(:first-child) th {
-      border-top: 1px solid $table-header-row-border-top-color;
-      background-color: darken($table-header-background-color, 3%);
-    }
-    tr:last-child th {
-      border-bottom: 1px solid $table-header-border-color;
-    }
-    tr:first-child {
+    tr {
       th {
         padding: 0 $space-xl 0 $space-m;
         line-height: $space-xl;
         white-space: nowrap;
         position: relative;
+        text-align: left;
+        border-bottom: 1px solid $table-header-border-color;
         label {
           font-size: rem(13px);
           font-weight: normal;
           text-transform: uppercase;
           display: inline-block;
+          vertical-align: top;
           margin: 0;
+          max-width: 100%;
+          overflow: hidden;
+          text-overflow: ellipsis;
         }
         .sort {
```

Comment:
Actually there is no sort component yet. The markup would looks like this in the bo fe where <template> could be exchanged with a simple <div>
```html
<template class="sort ${sort.field === key ? sort.direction : ''}">
    <i class="icon16 icon-sort-up icon-asc" click.delegate="change('asc')"></i>
    <i class="icon16 icon-sort-down icon-desc" click.delegate="change('desc')"></i>
</template>
```

the filter would be a ws-dropdown or ws-date-picker with an iconOnly flag. Check out the BO fe for that
https://github.bus.zalan.do/goodbuy/block-orders-frontend/blob/GB-863/src/app/view/overview/overview-grid.html